examples/kubernetes/simple/Dockerfile

```diff
@@ -1,8 +1,7 @@
-FROM python:3.5-slim-buster
+FROM optuna/optuna:py3.7-dev
 
 WORKDIR /usr/src/
 
-RUN pip install --no-cache-dir optuna scikit-learn psycopg2-binary
```

Comment:
I got the following error. I think `psycopg2-binary` is required to access PostgreSQL.

```
[E 2021-02-25 09:11:30,935] Failed to import DB access module for the specified storage URL. Please install appropriate one.  
```

Reply:
Changed the base image to `python:3.7-slim-buster` and spontaneously added back the `RUN` statement.